airbyte-cdk/bulk/core/load/src/main/kotlin/io/airbyte/cdk/load/data/json/JsonSchemaToAirbyteType.kt

```diff
@@ -10,7 +10,9 @@ import com.fasterxml.jackson.databind.node.ObjectNode
 import io.airbyte.cdk.load.data.*
 
```

Comment:
Some awkwardness introduced here by dropping the null type, but overall the class is slightly simpler, I think?